src/Aspire.Dashboard/Components/ResourcesGridColumns/LogLevelColumnDisplay.razor.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Aspire.Dashboard.Otlp.Model;
+using Aspire.Dashboard.Resources;
+using Microsoft.JSInterop;
+
+namespace Aspire.Dashboard.Components;
+
+public partial class LogLevelColumnDisplay
+{
+    private bool TryGetErrorInformation([MaybeNullWhen(false)] out string errorInfo)
+    {
+        if (LogEntry.Properties.GetValue("ex.Type") is { } exceptionType)
+        {
+            var message = LogEntry.Properties.GetValue("ex.Message");
+            var stackTrace = LogEntry.Properties.GetValue("ex.StackTrace");
+            errorInfo = LogEntry.Message + Environment.NewLine + $"{exceptionType}: {message}" + Environment.NewLine + stackTrace;
```

Comment:
```suggestion
            errorInfo = $"{exceptionType}: {message}" + Environment.NewLine + stackTrace;
```
Just include the exception information. There should be a separate way to copy the log message.

Reply:
Actually, from what I see of the data and according to the [spec](https://github.com/open-telemetry/semantic-conventions/blob/45a03e67ce2e109e860c38c60edf6367a1124545/docs/attributes-registry/exception.md), `exception.stacktrace` already includes the exception type and message.

errorInfo should be _just_ `exception.stacktrace`/`ex.StackTrace`.